index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" content"OOP test. Make use of constructor functions to create a fun Pirate website.">
+    <title>Pirate Bartender</title>
+    <link rel="stylesheet" href="main.css">
+    <script src="app.js"></script>
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"   integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s="   crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  </head>
+  <body>
+    <div class="container"> </div>
+      <h1>Pirate Bartender</h1>
+       <div class="questions"> </div>
+         <div class="userAnswer"> <input type="text"> </div>
+           <button id="submit">Sumbit your answer ye Landlubber!</button>
```

Comment:
Watch out for your indentation consistency. I would also highly suggest that you keep the HTML elements with the same amount of indentation unless you are putting them inside another element because it makes it easier to read. So I would do this:

<body>
     <div class="container"></div>
     <h1>Pirate Bartender</h1>
     <div class="questions"></div>
     <div class="userAnswer"><input type="text"></div>
     <button id="submit">Sumbit your answer ye Landlubber!</button>
</body>

On line 16 also, your "id" name "submit" is not very specific. I would suggest changing the name of this "id" to something more specific like "submitButton" or something along those lines.